packages/flutter_tool_macos/lib/main.dart

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
This might be a bit easier to bootstrap if this were kept in the flutter_tool package as a "builtin" extension. Could still spawn an isolate to run it, and disallow importing things it shouldn't import via tests.